docs/book/installation/sylius_plus_installation.rst

```diff
@@ -40,8 +40,10 @@ Installing Sylius Plus as a plugin to a Sylius application
 .. code-block:: bash
 
     composer config repositories.plus composer https://sylius.repo.packagist.com/ShortNameOfYourOrganization/
-    composer require sylius/plus --no-update
+    composer require symfony/dependency-injection
```

Comment:
Why this line is needed?

Reply:
![Uploading Screenshot 2021-07-05 at 12.53.27.png…]()